jarbas/core/migrations/0024_add_available_in_latest_dataset_field_to_reimbursement.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Two reasons:

* Because `last_update` is useful even if we're not using it for flagging stuff as `available_in_latest_dataset` or not
* Also it reflects the _baby steps_ methodology (as one can see in the [commits](https://github.com/datasciencebr/jarbas/pull/170/commits))